templates/web/src/permissions.ts.twig

```diff
@@ -0,0 +1,22 @@
+export class Permissions {
+
+    static read = (role: string[]): string => {
```

Comment:
All methods in this class should accept `string` instead of `string[]`

Reply:
done 👍